src/components/Tabs/Tabs.tsx

```diff
@@ -245,7 +245,7 @@ export const Tabs = ({
         // @ts-expect-error TODO: fix "No overload matches this call" error
         return React.cloneElement<Props>(child, {
           id: idVar[i],
-          ariaLabelledBy: ariaLabelledByVar[i],
+          ['aria-labelledby']: ariaLabelledByVar[i],
```

Comment:
Can you also update the other references to `areaLabelledBy` in this file?

Reply:
done